lib/ansible/modules/cloud/amazon/elasticache.py

```diff
@@ -14,6 +14,8 @@
 # You should have received a copy of the GNU General Public License
 # along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
 
+""" ansible elasticache module """
```

Comment:
You can delete this and the line below.
While here would you mind fixing the license http://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#documentation-block? It should look like:
```
#!/usr/bin/python
#
# Copyright (c) 2017 Ansible Project
#
# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
```